tests/baselines/reference/instanceofOnInstantiationExpression.errors.txt

```diff
@@ -0,0 +1,29 @@
+tests/cases/compiler/instanceofOnInstantiationExpression.ts(10,21): error TS2848: The right-hand side of an 'instanceof' expression must not be an instantiation expression.
+tests/cases/compiler/instanceofOnInstantiationExpression.ts(11,22): error TS2848: The right-hand side of an 'instanceof' expression must not be an instantiation expression.
+tests/cases/compiler/instanceofOnInstantiationExpression.ts(12,23): error TS2848: The right-hand side of an 'instanceof' expression must not be an instantiation expression.
+
+
+==== tests/cases/compiler/instanceofOnInstantiationExpression.ts (3 errors) ====
+    declare class Box<T> {
+        value: T;
+    }
+    
+    
+    declare const maybeBox: unknown;
+    
+    maybeBox instanceof Box; // OK
+    
+    maybeBox instanceof Box<number>; // error
+                        ~~~~~~~~~~~
+!!! error TS2848: The right-hand side of an 'instanceof' expression must not be an instantiation expression.
+    maybeBox instanceof (Box<number>); // error
+                         ~~~~~~~~~~~
+!!! error TS2848: The right-hand side of an 'instanceof' expression must not be an instantiation expression.
+    maybeBox instanceof ((Box<number>)); // error
+                          ~~~~~~~~~~~
+!!! error TS2848: The right-hand side of an 'instanceof' expression must not be an instantiation expression.
+    
+    Box<number> instanceof Object; // OK

```

Comment:
I am checking only the right side here, but, should these even be legal? They seem useless in another way.

Reply:
Is the instantiation expression pointless? Yes. Is it harmful? No. Is it misleading? Probably not really on the LHS, since the only reason I can think of why you'd want one on the LHS is to "satisfy" the RHS, but the RHS forbids them now, soooo...

Reply:
Yeah. I don't really care either way. I guess this PR is fine as-is.